### DISCUSSION TITLE
Json output mode

### DISCUSSION THREAD

**Discussion by lack** [ankitpokhrel/jira-cli#650] [Ideas]
I've just started using the tool, but keep finding weird corner cases in my personal workflow that I'd like to automate.

For this reason I'd love for the tool to generically support a `--json` flag which goes one step lower than the `--plain` to produce machine-readable json output.

I'm sure I could talk to the actual rest api myself, but this tool is so convenient for wrapping a bunch of ugly things so I don't have to worry about them, and it would be a powerful feature to open up the tool to script integration.

**Comment by php-coder** [comment]
See https://github.com/ankitpokhrel/jira-cli/discussions/256

**Comment by ankitpokhrel** [comment]
The upcoming release will include `--raw` for JSON output and additionally `--csv` for CSV output.
